mobile/library/common/http/client.cc

```diff
@@ -148,6 +151,22 @@ void Client::DirectStreamCallbacks::encodeData(Buffer::Instance& data, bool end_
   }
 }
 
+void Client::DirectStreamCallbacks::maybeEnableDataAvailableTimer() {
+  if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.report_available_data")) {
+    // Drain must have been called for us to get to the point we've received
+    // data from a server.
+    data_available_timer_ =
+        http_client_.dispatcher_.createTimerPostDrain([this]() { reportDataAvailable(); });
+    data_available_timer_->enableTimer(std::chrono::seconds(1), nullptr);
```

Comment:
should this be something like 5ms, so we know if we're immediately blocked on data?